doc/info-to-run-integration-tests-hermez-node.md

```diff
@@ -0,0 +1,34 @@
+# Run specifc hermez-node on integration-testing
+```
+Requirements: docker, docker-compose, nodejs v12.0+, aws cli 2.0+, git, jq
```

Comment:
If I am not wrong, they use nodejs v14 because it is the version that should use all the team. Also, npm v7 it is a requirement.

Reply:
Thank you for the suggestion!